contracts/position/PositionUtils.sol

```diff
@@ -710,14 +701,14 @@ library PositionUtils {
             params.position.isLong()
         );
 
-        return (priceImpactUsd, priceImpactAmount, sizeDeltaInTokens.toUint256(), executionPrice);
+        return (priceImpactUsd, priceImpactAmount, baseSizeDeltaInTokens, executionPrice);
```

Comment:
think we can keep the name as `sizeDeltaInTokens` instead of changing it to `baseSizeDeltaInTokens`